pkg/config/config.go

```diff
@@ -40,8 +45,13 @@ type Config struct {
 }
 
 func Default() *Config {
+	dir, err := ioutil.TempDir("", "dashboard-logs")
+	if err != nil {
+		log.Fatal("Failed to create directory for storing logs", zap.Error(err))
+	}
 	return &Config{
 		DataDir:            "/tmp/dashboard-data",
+		LogDir:             dir,
```

Comment:
Looks like we should assign `""` as default?

Reply:
Thanks for the advice.